packages/loader/container-loader/src/serializedStateManager.ts

```diff
@@ -185,6 +188,7 @@ export class SerializedStateManager {
 			);
 			this.snapshot = this.latestSnapshot;
 			this.latestSnapshot = undefined;
+			this.useNewSessionExpiryTime = true;
```

Comment:
Maybe we don't really need this member, but I prefer to be explicit, rather than inferring it from other conditions that could change as the code does.